cmd/testmev.go

```diff
@@ -0,0 +1,263 @@
+// Copyright © 2022-2024 Obol Labs Inc. Licensed under the terms of a Business Source License 1.1
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptrace"
+	"time"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/maps"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/app/z"
+)
+
+type testMEVConfig struct {
+	testConfig
+	Endpoints []string
+}
+
+type testCaseMEV func(context.Context, *testMEVConfig, string) testResult
+
+const (
+	thresholdMEVMeasureAvg = 40 * time.Millisecond
+	thresholdMEVMeasureBad = 100 * time.Millisecond
```

Comment:
@LukeHackett12 can I get your opinion on those 2 values? Those are the thresholds after which we consider average or bad performance for a RTT to a MEV relay.